src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    private static final Random random = new Random();
+    private final Color[] color = Color.values();
+
+    public static String getRandomColor() {
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index].name().toLowerCase();
```

Comment:
You're calling `Color.values()` twice which is not efficient. Since you already have an array `color` that holds the values, you should use it instead.